huqi.js.org

hu-qi opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (changes requested):
Hi there!

Your requested js.org subdomain must be inserted in alphabetical order in the `cnames_active.js` file before we can process your request. Please fix this. 🔤

hu-qi commented:
> Hi there!
> 
> Your requested js.org subdomain must be inserted in alphabetical order in the `cnames_active.js` file before we can process your request. Please fix this. 🔤

Thanks!

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```huqi.js.org``` to make it work. Right now it is pointing to "huqi.vercel.app"(https://github.com/hu-qi/hu-qi.github.io/blob/gh-page/CNAME)
I'll merge after we have "contact".

hu-qi commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `huqi.js.org` to make it work. Right now it is pointing to "huqi.vercel.app"(https://github.com/hu-qi/hu-qi.github.io/blob/gh-page/CNAME)
> I'll merge after we have "contact".

Thanks! I have updated the `CNAME`.

hu-qi commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `huqi.js.org` to make it work. Right now it is pointing to "huqi.vercel.app"(https://github.com/hu-qi/hu-qi.github.io/blob/gh-page/CNAME)
> I'll merge after we have "contact".

Thanks! I have updated the `CNAME`.

indus commented:
@hu-qi Looks like there is no content. Just a UmiJS template. Please fix this or postpone  your request until your page is ready for public.

indus commented:
No progress -closing.
